app/waf/src/controllers/application.controller.ts

```diff
@@ -299,96 +437,50 @@ export class ApplicationController {
     responses: {
       '204': Schema.emptyResponse('Successfully deploy Application resource'),
       '404': Schema.notFound('Can not find Application resource'),
+      '422': Schema.unprocessableEntity('Fail to deploy Application resource'),
     },
   })
   async deployById(
```

Comment:
Move to the declaration.controller is better since what users deploy is a declaration, not an application.